docs/pages/install.mdx

```diff
@@ -48,3 +39,44 @@ For example: 7.1.2
 - 2 represents a bug fix/patch release
 
 The download section above always shows the latest released version of Pelican and we recommend you to use the latest feature or major release version so it includes the latest bug fixes and features. If you would like to download a previous version, please refer to our [GitHub release page](https://github.com/PelicanPlatform/pelican/releases).
+
+## Install Pelican Binary
+
+### Linux
+
+* [Install Pelican on Red Hat Enterprise Linux (CentOS, Fedora, Alma, Rocky, or openSUSE)](./install/rhel.mdx)
+* [Install Pelican on Debian or Ubuntu](./install/debian.mdx)
+* [Install Pelican on Alpine Linux](./install/alpine.mdx)
+
+### macOS
+
+* [Install Pelican on macOS](./install/macos.mdx)
+
+## Verify Pelican Is Installed
+
+> If you installed Pelican as a standalone binary, you need to add Pelican to your `PATH` environment variable before proceeding.
+
+1. Run the following command after you installed Pelican RPM
+
+    ```bash
+    which pelican
+    ```
+
+    It should output
+
+    ```console
+    /bin/pelican
+    ```
+
+2. Test Pelican client function by running an **object copy** command that download a test file from [Open Science Data Federation (OSDF)](https://osg-htc.org/services/osdf) to your current directory
+
+    ```bash
+    $ pelican -f osg-htc.org object copy /ospool/uc-shared/public/OSG-Staff/validation/test.txt .
```

Comment:
I think I'm in favor of promoting pelican-style URLs wherever we can. Can you switch the command to:
```
$ pelican object copy pelican://osg-htc.org/ospool/uc-shared/public/OSG-Staff/validation/test.txt .
```

Reply:
I second this suggestion.